NOCList-ObjC/NOCList-ObjC/LSIAgent.h

```diff
@@ -0,0 +1,25 @@
+//
+//  LSIAgent.h
+//  NOCList-ObjC
+//
+//  Created by Karen Rodriguez on 5/10/20.
+//  Copyright © 2020 Hector Ledesma. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface LSIAgent : NSObject
+
+@property NSString *coverName;
+@property NSString *realName;
+@property NSNumber *accessLevel;
+@property NSNumber *compromised;
+
+- (instancetype)initWithCoverName:(NSString *)coverName realName:(NSString *)realName accessLevel:(int)accessLevel compromised:(BOOL)compromised;
```

Comment:
With these sorts of super-long lines, I like to split it up to make it a little bit more readable like so (like Paul suggested today):

```objc
- (instancetype)initWithCoverName:(NSString *)coverName
                         realName:(NSString *)realName
                      accessLevel:(int)accessLevel
                      compromised:(BOOL)compromised;
```